app/controllers/admin/auction_mark_payments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::AuctionMarkPaymentsController < Admin::BaseController
+  def update
+    @auction = Auction.find(params[:id])
+
+    paid_auction = MarkPaidAuction.new(
```

Comment:
I hear you, but this keeps it consistent with the other actions and it lets me test the functionality with a unit test